services/src/pro/handlers/users.rs

```diff
@@ -243,6 +251,73 @@ pub(crate) async fn session_view_handler<C: ProContext>(
     Ok(HttpResponse::Ok())
 }
 
+#[derive(Debug, Deserialize, Serialize, ToSchema)]
+pub struct QuotaUsed {
+    pub used: u64,
+}
+
+/// Retrieves the used quota of the current user.
+#[utoipa::path(
+    tag = "User",
+    get,
+    path = "/quota",
+    responses(
+        (status = 200, description = "The used quota of the user", body = QuotaUsed,
+            example = json!({
+                "used": 1234,
+            })
+        )
+    ),
+    security(
+        ("session_token" = [])
+    )
+)]
+pub(crate) async fn quota_handler<C: ProContext>(
+    ctx: web::Data<C>,
+    session: C::Session,
+) -> Result<impl Responder> {
+    let quota_used = ctx.user_db_ref().quota_used_by_session(&session).await?;
+    Ok(web::Json(QuotaUsed { used: quota_used }))
+}
```

Comment:
Coudln't we omit this API if the user knows his id?

Reply:
yes but it would be inconsistent with all the other APIs. We don't require the user id when getting the users project or something like that. As I see it, this API is for the user, and the other one is for an admin.

Reply:
then I would omit the `/qoutas/{userId}` one because for me it doesn't belong in the user API but some admin API which we do not have yet.

Reply:
But then we lose functionality, right?

Reply:
yes. I'd vote for keeping both methods and move the `/qoutas/{userId} ` into an admin API once we have one. We can also have a vote in the dev team which method to keep.

Reply:
But your admin API is hypothetical. There could also be different approaches. For instance, what if rights management allows project leaders to query the quotas of other team members? Should this be part of the admin API then?

Reply:
Shall I create a separate QuotaDb maybe for a) incrementing quota and b) accessing arbitrary user's quota? Then accessing your own quota goes through the UserDb and the rest to QuotaDb and we have no doubled functions.

Reply:
Nah, don't let us introduce some new type of DB where we do not know whether we need it.

It makes no difference now if we have this method in here on in another DB.
So just imagine you are in the QuotaDb ;-).

I made my point regarding redundancy. It is not wrong, so just decide what you want to go with.

Reply:
I'd go with the "redundancy" because to me it's two entirely different things, accessing your own quota and accessing other people's quota.